### ISSUE TITLE
what is landmark_location_x_1_sld... etc from final_csv.csv ?

### ISSUE THREAD

**Issue by essalim** [julialintern/fashionNet#2]
Hello, 
First of all, thank you for sharing this DeepFashion network implementation, i think this is the only implementation on the web, as I've been searching a lot for it in vain, the only remark I have is that the notebooks are quite hard to follow, as there are many missing chunks of code, for example, how did we get the values of landmark_location_x_1_sld... etc from final_csv.csv, a clean notebook would have been much more useful :)

**Comment by preetskhalsa97** [comment]
@eunjoa were you able to get an answer to the query?
If not, were you able to find another credible source for Fashion Net implementation?
